### PR TITLE
Added quality section

### PR DESCRIPTION
**Pull request by CathPag**

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview: https://deploy-preview-542--cncfglossary.netlify.app/style-guide/#quality-is-paramount

File: content/en/style-guide/_index.md [hunk 1]
```diff
@@ -105,6 +105,14 @@ The definitions for **technology** and **concept** categories contain three subh
 
 Note that **properties** don't require separate sections. A definition will suffice. 
 
+#### Quality is paramount
+
+If merged, your submission will be the official CNCF definition for that term (until someone else improves it). Creating a term that meets the CNCF's high standards can't be rushed — quality takes time and effort.
+
+**Do your research**: Even if you are confident you know the term, verify you go it right. We often use terms in organizations a certain way that may not reflect the full picture. When you do your research, especially when you're not 100% familiar with the term, use multiple resources. Many definitions are one-sided, especially if provided by a vendor. The Glossary must have vendor-neutral, globally accepted definitions.
```
Comment (iamNoah1) [inline content/en/style-guide/_index.md:112]:
```suggestion
**Do your research**: Even if you are confident you know the term, verify you got it right. We often use terms in organizations a certain way that may not reflect the full picture. When you do your research, especially when you're not 100% familiar with the term, use multiple resources. Many definitions are one-sided, especially if provided by a vendor. The Glossary must have vendor-neutral, globally accepted definitions.
```

File: content/en/style-guide/_index.md [hunk 2]
```diff
@@ -105,6 +105,14 @@ The definitions for **technology** and **concept** categories contain three subh
 
 Note that **properties** don't require separate sections. A definition will suffice. 
 
+#### Quality is paramount
+
+If merged, your submission will be the official CNCF definition for that term (until someone else improves it). Creating a term that meets the CNCF's high standards can't be rushed — quality takes time and effort.
+
+**Do your research**: Even if you are confident you know the term, verify you got it right. We often use terms in organizations a certain way that may not reflect the full picture. When you do your research, especially when you're not 100% familiar with the term, use multiple resources. Many definitions are one-sided, especially if provided by a vendor. The Glossary must contain vendor-neutral, globally accepted definitions.
+
+**Don't plagiarize**. The same rules apply to Glossary as to any other serious publication. Don't copy and paste other people's work unless you are quoting and contributing it to them. If you like a particular section of a definition, paraphrase it (describe it in your own words).
```
Comment (seokho-son) [inline content/en/style-guide/_index.md:114]:
```suggestion
**Don't plagiarize**. The same rules apply to Glossary as to any other serious publication. Don't copy and paste other people's work unless you are quoting and contributing it to them. If you like a particular section of a definition, paraphrase it in your own words.
```

**Review by seokho-son — COMMENTED** [review]
Could this change simplify the paragraph?

File: content/en/style-guide/_index.md (hunk 2 above)
Comment (seokho-son) [reply content/en/style-guide/_index.md:114]: applying this comment is up to you @CathPag :)

LGTM in advance.

**Review by seokho-son — APPROVED** [review]
LGTM

**Review by iamNoah1 — APPROVED** [review]
lgtm

**Pull request merged**